sccameraview/src/main/java/com/example/sccameraview/SaveImageTask.java

```diff
@@ -0,0 +1,64 @@
+package com.example.sccameraview;
+
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+import static com.example.sccameraview.BaseCameraView.LOG_TAG;
+
+class SaveImageTask extends AsyncTask {
+
+    OnImageSavedListener imageSavedListener;
+    private byte[] imageData;
```

Comment:
We should pass all parameters in the constructor, and make these fields `final`.